test/board_test.exs

```diff
@@ -2,16 +2,22 @@ defmodule BoardTests do
   use ExUnit.Case
 
   test "can create a board" do
-    board = Board.create(3)
+    assert Board.create == [["1", "2", "3"], ["4", "5", "6"],[ "7", "8", "9"]]
+  end
+
+  test "can place a marker on the bord" do
+    board = Board.create
+    changed_board = Board.place_marker(6, board, :x)
 
-    assert board =={:x, [["1", "2", "3"], ["4", "5", "6"],[ "7", "8", "9"]], :first_turn}  
+    assert changed_board == [["1","2","3"],["4","5","X"],["7","8","9"]]
   end
 
-  test "can make a move" do
-    board = Board.create(3)
-    
-    changed_board = Board.place_marker(6,board)
+  test "it can detect if a tile has already been taken" do
+    assert Board.check_tile_not_taken([["1","2","3"],["4","5","X"],["7","8","9"]], 6) == :tile_already_selected
+  end
```

Comment:
There doesnt seem that many Board tests for the length of the module. There should be at least one unit test for every public method. I don't see one for `create_next_game_status`, `create_next_game_status`, `get_marker_symbol` etc.. Whilst they may be tested through other modules, its good to make the unit tests complete so that should an error occur, a very focussed test fails, leading to easy diagnosis and resolution.